src/index.html

```diff
@@ -7,12 +7,55 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="product_image"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <div
+        class="card__code"
+        data-qa="card"
+      >
+        <p class="card__code-text">Product code:</p>
+        <p class="card__code-number">195434</p>
+      </div>
+      <div class="card__stars-reviews-wrapper">
+        <div class="card__stars card__stars--4">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+        <div class="card__reviews">
+          <p class="card__reviews-text">Reviews:</p>
+          <p class="card__reviews-number">5</p>
+        </div>
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-number">$2,199</p>
+      </div>
+      <button
+        class="card__button"
+        data-qa="hover"
```

Comment:
The 'data-qa' attribute with the value 'hover' seems to be a testing attribute. Verify that it is necessary for the project requirements, otherwise, it should be removed.